app/src/main/java/com/github/onlynotesswent/ui/overview/FolderContent.kt

```diff
@@ -375,7 +418,14 @@ fun FolderContentTopBar(
               expanded = expanded,
               onFabClick = { onExpandedChange(true) },
               onDismissRequest = { onExpandedChange(false) })
+        } else if (folder.visibility == Visibility.PUBLIC ||
```

Comment:
Shouldn't we instead check that if the folder is public, the owner shouldn't be the current user ?

Reply:
We already know they aren't from the previous if, however I did change to use the new .isVisibleTo function which was a very good recommendation to simplify

Reply:
Check done before (same as Edit Note comment)

Reply:
Okay, great